<launch>

  <node pkg="laserScan_Similarity" type="laserScan_Similarity_kitti_Rings" name="laserScan_Similarity_kitti_Rings" output="screen" >

    <param name="inputQueueSize" value="1"/>
    <param name="rangeThreshold" value="0.0"/>
    <param name="isVoxelFilter" value="false"/>
    <param name="voxelFilterConfig" value="$(find laserScan_Similarity)/cfg/voxel_filter.yaml" />

    <param name="sectionNumOfR" value="40"/>
    <param name="minRange" value="0.0"/>
    <param name="maxRange" value="80.0" />

    <param name="sectionNumOfI" value="20"/>
    <param name="minIntensity" value="0.0"/>
    <param name="maxIntensity" value="150.0"/>

    <param name="baseDirKitti" value="/home/yh/kitti_benchMark/00/velodyne_points/data/"/>

  </node>
</launch>
